Index: package/base-files/files/sbin/sysupgrade
===================================================================
--- package/base-files/files/sbin/sysupgrade
+++ package/base-files/files/sbin/sysupgrade
@@ -101,7 +101,7 @@
 add_uci_conffiles() {
 	local file="$1"
 	( find $(sed -ne '/^[[:space:]]*$/d; /^#/d; p' \
-		/etc/sysupgrade.conf /lib/upgrade/keep.d/* 2>/dev/null) \
+		/etc/sysupgrade.conf 2>/dev/null) \
 		-type f -o -type l 2>/dev/null;
 	  opkg list-changed-conffiles ) | sort -u > "$file"
 	return 0
