Index: target/linux/ramips/base-files/etc/uci-defaults/network
===================================================================
--- target/linux/ramips/base-files/etc/uci-defaults/network	(Revision 26472)
+++ target/linux/ramips/base-files/etc/uci-defaults/network	(Arbeitskopie)
@@ -1,5 +1,5 @@
 #!/bin/sh
-RT305X=`cat /proc/cpuinfo | grep RT305`
+RT305X=`cat /proc/cpuinfo | grep RT3.5`
 [ -z "${RT305X}" ] || {
 	uci batch <<EOF
 set network.lan.ifname=eth0.1
