Index: build_dir/target/linux/ramips/rt305x/config-2.6.39
===================================================================
--- build_dir/target/linux/ramips/rt305x/config-2.6.39	(Revision 28601)
+++ build_dir/target/linux/ramips/rt305x/config-2.6.39	(Arbeitskopie)
@@ -1,3 +1,4 @@
+CONFIG_RFKILL=y
 # CONFIG_ARCH_DMA_ADDR_T_64BIT is not set
 # CONFIG_ARCH_HAS_ILOG2_U32 is not set
 # CONFIG_ARCH_HAS_ILOG2_U64 is not set
