From: =?UTF-8?q?Fabian=20Bl=C3=A4se?= <fabian@blaese.de>
Date: Mon, 6 Jan 2025 20:52:13 +0100
Subject: [PATCH] mt7621: bump compat version

diff --git a/target/linux/ramips/image/mt7621.mk b/target/linux/ramips/image/mt7621.mk
index eb7dacefef..af91d26af4 100644
--- a/target/linux/ramips/image/mt7621.mk
+++ b/target/linux/ramips/image/mt7621.mk
@@ -2105,6 +2105,8 @@ endef
 
 define Device/netgear_r6220
   $(Device/netgear_sercomm_nand)
+  DEVICE_COMPAT_VERSION := 2.0
+  DEVICE_COMPAT_MESSAGE := Config cannot be migrated from swconfig to DSA
   DEVICE_MODEL := R6220
   SERCOMM_HWNAME := R6220
   SERCOMM_HWID := AYA
@@ -3132,6 +3134,8 @@ TARGET_DEVICES += xiaomi_mi-router-4
 define Device/xiaomi_mi-router-4a-gigabit
   $(Device/dsa-migration)
   $(Device/uimage-lzma-loader)
+  DEVICE_COMPAT_VERSION := 2.0
+  DEVICE_COMPAT_MESSAGE := Config cannot be migrated from swconfig to DSA
   IMAGE_SIZE := 14848k
   DEVICE_VENDOR := Xiaomi
   DEVICE_MODEL := Mi Router 4A
@@ -3143,6 +3147,8 @@ TARGET_DEVICES += xiaomi_mi-router-4a-gigabit
 define Device/xiaomi_mi-router-4a-gigabit-v2
   $(Device/dsa-migration)
   $(Device/uimage-lzma-loader)
+  DEVICE_COMPAT_VERSION := 2.0
+  DEVICE_COMPAT_MESSAGE := Config cannot be migrated from swconfig to DSA
   IMAGE_SIZE := 14784k
   DEVICE_VENDOR := Xiaomi
   DEVICE_MODEL := Mi Router 4A

