Index: include/image.mk
===================================================================
--- include/image.mk	(Revision 35298)
+++ include/image.mk	(Arbeitskopie)
@@ -42,10 +42,10 @@
   JFFS2OPTS += -x lzma
 endif
 
-SQUASHFS_BLOCKSIZE := 256k
+SQUASHFS_BLOCKSIZE := 64k
 SQUASHFSOPT := -b $(SQUASHFS_BLOCKSIZE)
 SQUASHFSCOMP := gzip
-LZMA_XZ_OPTIONS := -Xpreset 9 -Xe -Xlc 0 -Xlp 2 -Xpb 2
+LZMA_XZ_OPTIONS := -Xpreset 1
 ifeq ($(CONFIG_SQUASHFS_LZMA),y)
   SQUASHFSCOMP := lzma $(LZMA_XZ_OPTIONS)
 endif
